mozregression/bisector.py

```diff
@@ -531,13 +531,8 @@ def bisect_nightlies(self, good_date, bad_date):
     def bisect_inbound(self, good_rev, bad_rev):
         self._logger.info("Getting inbound builds between %s and %s"
                           % (good_rev, bad_rev))
-        # anything within twelve hours is potentially within the range
-        # (should be a tighter but some older builds have wrong timestamps,
-        # see https://bugzilla.mozilla.org/show_bug.cgi?id=1018907 ...
-        # we can change this at some point in the future, after those builds
-        # expire)
         handler = InboundHandler(find_fix=self.options.find_fix)
-        result = self.do_bisect(handler, good_rev, bad_rev, range=60*60*12)
+        result = self.do_bisect(handler, good_rev, bad_rev)
```

Comment:
So as discussed on irc, on the ftp the builds were indexed by build timestamp - so we needed that code to find every build folder. This is no more usefull with taskcluster, since it indexes on changesets directly. The code that handle the move from nightly to inbound (going back in time 4 days ago, to not miss changesets) is still there.